Makefile

```diff
@@ -1,43 +1,35 @@
-all : .state_xnat-web-lxd .state_xnat-web-docker
+xnat_version = 1.8.1
+xnat_ver_17 = 1.7.6
+plugin_ldap_auth_ver = 1.1.0
+plugin_openid_auth_ver = 1.0.2
 
-xnat_web_ver = 1.7.6
-XNAT := xnat.json setenv.sh xnat-web-1.7.6.war openid-auth-plugin-1.0.0.jar xnat-ldap-auth-plugin-1.0.0.jar
+xnat_war = xnat-web-$(xnat_version).war
+xnat_url = https://bitbucket.org/xnatdev/xnat-web/downloads/$(xnat_war)
+xnat_war_17 = xnat-web-$(xnat_ver_17).war
+xnat_url_17 = https://bitbucket.org/xnatdev/xnat-web/downloads/$(xnat_war_17)
+plugin_ldap_auth_jar = ldap-auth-plugin-$(plugin_ldap_auth_ver).jar
+plugin_ldap_auth_url = https://bitbucket.org/xnatx/ldap-auth-plugin/downloads/ldap-auth-plugin-1.1.0.jar
+plugin_openid_auth_jar = openid-auth-plugin-$(plugin_openid_auth_ver).jar
+plugin_openid_auth_url = https://github.com/Australian-Imaging-Service/xnat-openid-auth-plugin/releases/download/$(plugin_openid_auth_ver)/xnat-openid-auth-plugin-all-$(plugin_openid_auth_ver).jar
```

Comment:
The OpenID plugin doesn't support XNAT 1.8. Perhaps check for this, or just documenting it should be fine.

Reply:
Would it be acceptable to append -rc 'release candidate' to the image at this time to show that it is under development and not production.

Reply:
Works for me!